File: python/questao10.py
def main():
    consumo = float(input("Qual o consumo? "))
    installation = input("Qual o tipo de residência? (R para residências, I para indústrias e C para comércios) ")

    if installation == 'R':
        price = consumo * 0.4 if consumo <= 500 else consumo * 0.65
    elif installation == 'I':
        price = consumo * 0.55 if consumo <= 1000 else consumo * 0.6
    elif installation == 'C':
        price = consumo * 0.55 if consumo <= 5000 else consumo * 0.6
    else:
        print("Opção de instalação inválida!")
        return

    print(f"O valor a pagar será R$ {price:.2f}")

if __name__ == "__main__":
    main()

File: c/questao10.c
#include <stdio.h>

int main() {
    float price, consumo;
    char installation;

    printf("Qual o consumo? ");
    scanf("%f", &consumo);

    printf("Qual o tipo de residência? (R para residências, I para indústrias e C para comércios) ");
    scanf("%s", &installation);

    switch (installation)
    {
    case 'R':
        if (consumo <= 500) {
            price = consumo * 0.4;
        } else {
            price = consumo * 0.65;
        }
        break;
    case 'I':
        if (consumo <= 1000) {
            price = consumo * 0.55;
        } else {
            price = consumo * 0.6;
        }
        break;
    case 'C':
        if (consumo <= 5000) {
            price = consumo * 0.55;
        } else {
            price = consumo * 0.6;
        }
        break;
    default:
        printf("Opção de instalação inválida!");
        break;
    }

    printf("O valor a pagar será R$ %.2f\n", price);


    return 0;
}